src/utils/NodeConnection.js

```diff
@@ -31,6 +31,7 @@ define(function (require, exports, module) {
     /**
```

Comment:
Remove this file from API docs as `NodeConnection.js` is deprecated and wont work in phoenix. Its just kept there for compatibility with legacy extensions.